package.json

```
@@ -0,0 +1,69 @@
+{
+  "version": "0.1.0",
+  "license": "MIT",
```

```suggestion
  "license": "Apache-2.0",
```

once the actual LICENSE file changes